modules/utils/docker/entrypoint.sh

```diff
@@ -98,7 +192,20 @@ elif [ "$mode" = "ap+mesh_mcc" ]; then
   # Set frequency band and channel from given frequency
   calculate_wifi_channel "$ch"
   ifconfig $ifname_ap up
+  
+  #random generate the ipv6
+  IPV6_PREFIX=$( echo fd`dd if=/dev/urandom bs=7 count=1 status=none | xxd -p` | sed 's/\(....\)/\1:/g' )
+  ip addr add $SUBNET.1/24 dev br-lan && ip addr add $IPV6_PREFIX:1/64 dev br-lan
+
+  create_olsrd_config "wlp1s0" "$SUBNET" "$IPV6_PREFIX"
```

Comment:
We can move  olsr config update under  "$algo" = "olsr" , may be this can go in next PR.